src/content.ts

```diff
@@ -99,6 +99,12 @@ export type Paragraph = {
    * whose size is the maximum of the two margins.
    */
   margin?: Length | BoxLengths;
+  /**
+   * Align text in a paragraph.
+   * textAlign supports left, right and center alignment.
```

Comment:
You don't need to repeat the attribute name here. "Supports left, right and center." is sufficient. Preferably, put the literals in backticks.